go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/OctopusDeploy/cli
 
 go 1.19
 
+replace github.com/OctopusDeploy/go-octopusdeploy/v2 => ../go-octopusdeploy
```

Comment:
NOTE: I promise not to merge this.
The reason I've left it in is because the CLI depends on the yet-to-be-merged PR from go-octopusdeploy.

After that merges, we'll need to create a new release (2.x.x) then update this go mod file to point to that instead, THEN this PR can merge